kalite/settings/__init__.py

```diff
@@ -148,11 +145,12 @@ def package_selected(package_name):
     # use another sqlite3 database for the screenshots
     DATABASES = {
         SCREENSHOTS_ROUTER: {
-            "ENGINE": getattr(local_settings, "DATABASE_TYPE", SQLITE3_ENGINE),
-            "NAME": SCREENSHOTS_DATABASE_PATH,
-            "OPTIONS": {
-                "timeout": 60,
-            },
-        }
+            "ENGINE": SQLITE3_ENGINE,
```

Comment:
Rather than another `if` statement in our settings file, perhaps we should move to @benjaoming's `--settings` arg method instead.